certified-connectors/LAWLIFT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,660 @@
+              "x-ms-visibility": "advanced"
+            },
+            "htmlMessage": {
+              "type": "string",
+              "description": "LAWLIFT generated HTML email message.",
+              "title": "HTML Email Message",
+              "x-ms-visibility": "important"
+            },
+            "plainMessage": {
+              "type": "string",
+              "description": "LAWLIFT generated plain text email message.",
+              "title": "Plain Text Email Message",
+              "x-ms-visibility": "advanced"
+            },
+            "answersJson": {
+              "format": "byte",
+              "type": "string",
+              "title": "Selected Answers JSON",
+              "description": "Content of the selected answers JSON file.",
+              "x-ms-visibility": "advanced"
+            },
+            "answersCsv": {
+              "format": "byte",
+              "type": "string",
+              "title": "Selected Answers CSV",
+              "description": "Content of the selected answers CSV file.",
+              "x-ms-visibility": "advanced"
+            },
+            "folderPath": {
+              "type": "string",
+              "description": "Default /lawlift folder path.",
+              "title": "Folder Path",
+              "x-ms-visibility": "advanced"
+            },
+            "metadata": {
+              "type": "object",
+              "properties": {
+                "publication": {
+                  "type": "object",
+                  "x-schema": {
+                    "$ref": "#/definitions/PublicationMetadata"
+                  }
+                },
+                "template": {
+                  "type": "object",
+                  "x-schema": {
+                    "$ref": "#/definitions/TemplateMetadata"
+                  }
+                }
+              },
+              "title": "Metadata",
+              "description": "Current flow metadata.",
+              "x-ms-visibility": "advanced"
+            }
+          }
+        },
+        "description": "LAWLIFT trigger for sending notifications."
+      },
+      "post": {
+        "tags": [
+          "Send Notifications"
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "The message to be displayed to the flow initiator."
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request"
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        },
+        "security": [
+          {
+            "API Key": []
+          }
+        ],
+        "summary": "LAWLIFT notification trigger",
```

Comment:
@makaveli1313 Usually summaries of triggers would start with "When .." or "Trigger..", Can you please update it